frontend/src/constants/routes.ts

```diff
@@ -42,8 +42,8 @@ const ROUTES = {
 	TRACE_EXPLORER: '/trace-explorer',
 	BILLING: '/billing',
 	SUPPORT: '/support',
-	LOGS_SAVE_VIEWS: '/logs-save-views',
-	TRACES_SAVE_VIEWS: '/traces-save-views',
+	LOGS_SAVE_VIEWS: '/logs/saved-views',
+	TRACES_SAVE_VIEWS: '/traces/saved-views',
```

Comment:
Please add old - new route mapping to handle auto redirects for saved / old routes

Reply:
Done.